models/receipt.go

```diff
@@ -22,26 +22,68 @@ import (
 // geth node has the data locally, but we don't in evm gateway, so we can not reproduce those values
 // and we need to store them
 type StorageReceipt struct {
-	Type              uint8
-	PostState         []byte
-	Status            uint64
-	CumulativeGasUsed uint64
+	Type              uint8  `json:"type,omitempty"`
+	PostState         []byte `json:"root"`
+	Status            uint64 `json:"status"`
+	CumulativeGasUsed uint64 `json:"cumulativeGasUsed"`
 	// todo we could skip bloom to optimize storage and dynamically recalculate it
-	Bloom             gethTypes.Bloom
-	Logs              []*gethTypes.Log
-	TxHash            common.Hash
-	ContractAddress   common.Address
-	GasUsed           uint64
-	EffectiveGasPrice *big.Int
-	BlobGasUsed       uint64
-	BlobGasPrice      *big.Int
-	BlockHash         common.Hash
-	BlockNumber       *big.Int
-	TransactionIndex  uint
+	Bloom             gethTypes.Bloom  `json:"logsBloom"`
+	Logs              []*gethTypes.Log `json:"logs"`
+	TxHash            common.Hash      `json:"transactionHash"`
+	ContractAddress   common.Address   `json:"contractAddress"`
+	GasUsed           uint64           `json:"gasUsed"`
+	EffectiveGasPrice *big.Int         `json:"effectiveGasPrice"`
+	BlobGasUsed       uint64           `json:"blobGasUsed,omitempty"`
+	BlobGasPrice      *big.Int         `json:"blobGasPrice,omitempty"`
+	BlockHash         common.Hash      `json:"blockHash,omitempty"`
+	BlockNumber       *big.Int         `json:"blockNumber,omitempty"`
+	TransactionIndex  uint             `json:"transactionIndex"`
+	RevertReason      []byte           `json:"revertReason"`
+}
+
+func (sr *StorageReceipt) ToGethReceipt() *gethTypes.Receipt {
+	return &gethTypes.Receipt{
```

Comment:
If above can be done then we can do
```suggestion
	return sr.Receipt
```